docs/topics/js/js-get-started.md

```diff
@@ -102,21 +106,19 @@ Enter your name in the text box and accept the greetings from your application!
 
 1. Open the file `welcome.kt` in `src/main/kotlin`.  
 
-2. Add a `button` element with an `onClickFunction` event handler.  
+2. Add a `button` element with an `onClick` event handler.  
    
-   > Make sure that you import the package `kotlinx.html.js.*`.
+   > Make sure that you import the `react.dom.html.ReactHTML.button` element.
```

Comment:
Not sure if this is clear, these are the suggestions I'm getting

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/2178959/164456063-a1e19a2e-0aeb-44a7-8b2d-59b1cb7dbc5c.png">

Reply:
Looks good!